engine/table/src/test/java/io/deephaven/engine/table/impl/ssa/TestByteSegmentedSortedArray.java

```diff
@@ -66,8 +66,12 @@ public void testInsertAndRemove() {
 
     public void testShifts() {
         final SsaTestHelpers.TestDescriptor desc = new SsaTestHelpers.TestDescriptor();
-        for (int seed = 0; seed < 20; ++seed) {
-            for (int tableSize = 10; tableSize <= 10000; tableSize *= 10) {
+        final int nSeeds = scaleToDesiredTestLength(20);
+        final int tableSizeSteps = scaleToDesiredTestLength(1000);
+        final int maxTableSize = 10000;
+        final int tableSizeStep = maxTableSize / tableSizeSteps;
```

Comment:
I think "Step" is a misnomer - in my mind, it implies equally-spaced additions. I think "Mult" may be better?

Reply:
I actually did not notice int this case this was a factor instead of an additive step; thanks.